docs/reference/export-metrics.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Exporting Metrics (Prometheus)
+sidebar_position: 1200
+description: Using OpenTelemetry to export gadget metrics to Prometheus
+---
+
+If a gadget is able to collect metrics ([read here](../gadget-devel/metrics) to learn more about enabling metrics for
+your own gadgets]), you can easily expose those to a Prometheus endpoint. A gadget that can export metrics should have
+at least one of its data sources annotated with the `metrics.collect=true` annotation.
```

Comment:
There're no metadata files with `metrics.collection=true` 🙁. How can users check that?